recipes/apriltag/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "apriltag" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://github.com/AprilRobotics/apriltag/archive/refs/tags/v{{ version }}.tar.gz
+  # sha256: 68ef6640f55aac6566ba8d030a4e7d960446d5b9340dfb61e79b9f92a08b1032
+  git_url: https://github.com/AprilRobotics/apriltag.git
+  git_rev: master
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - pkg-config
+    - ninja
+    - python
+    - numpy
+  host:
+    - python
+    - numpy
+  run:
+    - python
+
+test:
+  requires:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+    - python
+  files:
+    - CMakeLists.txt
+    - test.c
+  commands:
+    - cmake -G Ninja -B build_test
+    - cmake --build build_test
+    - python -c "from apriltag import apriltag"  # [linux]
```

Comment:
Why are you only testing this on linux?

Reply:
On macOS I got a segfault on the CI and I don't have the hardware to debug this. On Windows I disabled the Python bindings in the hope of fixing the second issue I reported above (`Empty package; python present in build and host deps.`).

Reply:
I am keeping this as the Windows build fails to detect Python and NumPy, even though they are specified and detected by the other OSes, and because the macOS test segfaults. As I do not have macOS hardware, I cannot debug the issue and will leave this to someone who has the hardware.

Reply:
Please enable win and mac again. I can then look at the failures. They might be simple to fix given the right knowledge; but I would need to see the build logs to help you.

Reply:
Thanks for helping out with this. I removed the "linux" flag and enabled the Python tests for all targets.

Reply:
I reset the commits in order to continue with this release.